### PR TITLE
Ana/add default lunie theme

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Adds a default Lunie theme (lunie-dark) to stay safe and also because it is needed to integrate new networks.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="app" :class="network">
+  <div id="app" :class="theme">
```
Comment (faboweb) [inline src/App.vue:2]: @jbibla can we just do this?:

```suggestion
  <div id="app" class="lunie-dark" :class="network">
```

Comment (Bitcoinera) [reply src/App.vue:2]: I just tried it and it only picks `lunie-dark`

**Comment by Bitcoinera** [comment]
Wow, unit tests were failing from my prior PR with Dawn. How come they didn't fail there?

File: src/App.vue [hunk 2]
```diff
@@ -38,10 +38,16 @@ export default {
   },
   data: () => ({
     isMobileApp: config.mobileApp,
+    networksThemes: ["cosmos", "terra", "emoney", "kava", "kusama"],
```
Comment (faboweb) [inline src/App.vue:41]: I am a little worried, that this doesn't scale

Comment (Bitcoinera) [reply src/App.vue:41]: How to do it then? CSS is quite dumb

Comment (Bitcoinera) [reply src/App.vue:41]: OK, so what's the plan Stan

Comment (faboweb) [reply src/App.vue:41]: I talked with @jbibla to experiment with default variables. We could just set default variables (lunie-dark) as a default and overwrite these variables with the network class

Comment (Bitcoinera) [reply src/App.vue:41]: Sounds good. Unsure how to implement that without a similar logic than what we have in this PR (array with current networks slugs)

Comment (faboweb) [reply src/App.vue:41]: how about setting .lunie-dark on #app?

Comment (Bitcoinera) [reply src/App.vue:41]: just tried. It doesn't work

**Comment by Bitcoinera** [comment]
OK, why not do this? I mean, to work with Akash I need this. I think it is also safer to have this logic.

I also think it is scalable. I can change the Array to a "networks" query in App.vue and then it is updated automatically.

How does that sound?

**Comment by Bitcoinera** [comment]
Probably should have done the networks query thing from the beginning actually

**Comment by Bitcoinera** [comment]
Another alternative is to add a new field in the networks Object, `hasTheme`, a Boolean

**Comment by faboweb** [comment]
solution: add .lunie-dark to body

**Comment by Bitcoinera** [comment]
> solution: add .lunie-dark to body

Yes, nice, that was it!

Only some minor CSS issues in lunie-dark... 👁️ 

image

File: src/components/common/AppMenu.vue [hunk 3]
```diff
@@ -94,7 +94,7 @@
       </router-link>
 
       <router-link
-        v-if="session.experimentalMode"
+        v-if="session.developmentMode"
```
Comment (faboweb) [inline src/components/common/AppMenu.vue:97]:
```suggestion
        v-if="session.experimentalMode"
```

**Pull request merged**